### PR TITLE
Use i18n strings in codebytes banner and add English only descriptor

### PR DESCRIPTION
**Pull request by dju90**
...and added strings to [pegasus gsheet](https://docs.google.com/spreadsheets/d/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU/edit#gid=0)

## Testing story
Tested on localhost
_English_
image

_Spanish_
image

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dju90** [comment]
@molly-moen any ideas on how to apply the original link color style (#7665a0) to the codebytes url without splitting up that string?

**Comment by molly-moen** [comment]
@dju90 could you add css that applies to an 'a' tag inside some other tag (maybe add a class around that string)? [Example of css in haml](https://github.com/code-dot-org/code-dot-org/blob/d9f8d3e87a7e207346bb4ad6b2e2ae4b43098978/pegasus/sites.v3/code.org/views/homepage_hero.haml#L133)

**Comment by dju90** [comment]
@molly-moen Ooh, cool. I just tried it though, and it looks like the css is being overridden by another style...
image

**Review by molly-moen — APPROVED** [review]
Would be good to double check that the css doesn't overwrite any other a tags on the page, otherwise LGTM!

File: pegasus/sites.v3/code.org/views/codebytes_skinnybanner.haml [hunk 1]
```diff
@@ -4,11 +4,15 @@
...
+      = I18n.t(:codebytes_skinny_banner_title)
     %p{style:"font-size: 14px; border: 0; color: #4d575f"}
-      Join us each day of CS Education Week for CodeBytes, 20 minute interactive computer science lessons. Streaming live twice daily, CodeBytes is a great way for students to participate in a virtual #HourOfCode!
-    %p{style:"font-size: 14px; border: 0; color: #4d575f"}
-      Learn more at
-      %a{href:"https://code.org/codebytes", target: '_blank', style:"color:#7665a0"}
-        code.org/codebytes
+      = I18n.t(:codebytes_skinny_banner_description)
+      = I18n.t(:codebytes_skinny_banner_english_only)
+    %p{style:"font-size: 14px; border: 0; color: #4d575f", class:"skinny-banner-link"}
```
Comment (molly-moen) [inline pegasus/sites.v3/code.org/views/codebytes_skinnybanner.haml]: nit: instead of adding the class manually, you can do `%p.skinny-banner-link`

**Pull request merged**